### PR TITLE
[TASK] Add missing compression support in RedisBackend

### PR DESCRIPTION
**Pull request by dfeyer**
Resolves: FLOW-420

### CODE REVIEW

File: TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/RedisBackend.php [hunk 1]
```diff
@@ -415,6 +416,43 @@ public function setDatabase($database)
     }
 
     /**
+     * @param integer $compressionLevel
```
Comment (aertmann) [inline TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/RedisBackend.php]: @return void

**Comment by aertmann** [comment]
:+1: by reading aside from missing return

**Comment by robertlemke** [comment]
@dfeyer good catch!

**Comment by albe** [comment]
:+1: by reading, but would it maybe make sense to add a compressionThreshold option like in memcache, so that only data larger than X bytes is compressed?

**Comment by robertlemke** [comment]
@albe yes, good idea. In that case we might also enable compression by default, because we'll have to unzip the data when reading any time then.

**Comment by kitsunet** [comment]
IMHO we could make that two separate commits, first introducing compression with this commit and then adding a threshold and automation in a separate commit as that is more complicated and IMHO needs careful measurements of the speed benefit vs. overhead.

**Comment by kitsunet** [comment]
that said :+1: for this commit besides askes comment which is valid but wouldn't be a blocker for merge for me :)

**Pull request merged**